### ISSUE TITLE
Add Inline Numeric Slider

### ISSUE THREAD

**Issue by mikethedj4** [chinchang/web-maker#167]
I know [Inlet.js](https://github.com/enjalot/Inlet) comes with a built in numeric slider. I was wondering why this hasn't been integrated into webMaker?

**Comment by chinchang** [comment]
You are right, its there by default in Inlet.js. But I disabled it after receiving feedback and also myself realising that it has very poor user experience. Eg. it shows up unintentionally a lot of times, creating huge UNDO stacks on little sliding etc.

**Comment by mikethedj4** [comment]
Why not at least have it as an option for the CSS editor? Just makes since.
